recipe/meta.yaml

```diff
@@ -13,45 +13,34 @@ source:
   sha256: c57ce44873abcfcfd1d61d7d261e416d352186958e7b5d299cf244efa6757816
 
 build:
-  number: 2
-  script_env:
-    - F2C_EXTERNAL_ARITH_HEADER={{ RECIPE_DIR }}/arith_arm64.h  # [arm64]
-
-  run_exports:
-    - {{ pin_subpackage("python-igraph", max_pin="x.x") }}
+  number: 3
 
 requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - libtool  # [unix]
     - {{ compiler('c') }}
     - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
-    # just to make cmake happy
-    - bison  # [osx and arm64]
-    - cmake
-    - ninja
-    - make  # [not win]
-    - pkg-config  # [not win]
+    - pkg-config
   host:
-    - glpk
-    - libxml2-devel
-    - arpack   # [not win]
-    - gmp
-    - libblas
-    - liblapack
     - python
     - pip
     - setuptools >=64,<72.2.0
-    - zlib
+    - igraph >=0.10,<0.11
```

Comment:
Based on @szhorvat's comment: https://github.com/conda-forge/python-igraph-feedstock/pull/97#discussion_r2384178009, we may need this to be pinned to the patch version in both host and run, and ignore the run exports.